PID:
  Kp_x : 10.5
  Kp_y : 10.5
  Kp_z : 6.5
  Kp_vx : 0.2
  Kp_vy : -0.2
  Kp_vz : 1.8

  Ki_x : 0.2
  Ki_y : 0.08
  Ki_z : 0.08
  Ki_vx : 0.0
  Ki_vy : 0.0
  Ki_vz : 0.0

  Kd_x : 0
  Kd_y : 0
  Kd_z : 0
  Kd_vx : 0
  Kd_vy : 0
  Kd_vz : 0
  K_yaw : 6.0

  position_x_i_ub : 5.0
  position_x_i_lb : -5.0
  position_y_i_ub : 5.0
  position_y_i_lb : -5.0
  position_z_i_ub : 5.0
  position_z_i_lb : -5.0
  velocity_x_i_ub : 0.8
  velocity_x_i_lb : -0.8
  velocity_y_i_ub : 0.8
  velocity_y_i_lb : -0.8
  velocity_z_i_ub : 0.8
  velocity_z_i_lb : -0.8
  thrust_bound : 0.3
  x_bound : 0.5
  y_bound : 0.5

Lowpass_filter:
  lp3_k : 0.2

Drone:
  mass : 0.305
  g : 9.80665
  balance_thrust : 0.0
